flink-connector-debezium/src/main/java/com/alibaba/ververica/cdc/debezium/DebeziumSourceFunction.java

```diff
@@ -303,6 +311,9 @@ public void run(SourceContext<T> sourceContext) throws Exception {
 		// run the engine asynchronously
 		executor.execute(engine);
 
+		// run the real debezium consumer
+		executor.execute(debeziumConsumer);
```

Comment:
We don't need to start a new thread to poll elements from the queue. We can poll in the following while loop.

Reply:
OK, I will modify this.

Reply:
done